src/main/java/net/minecraftforge/client/renderer/transparency/OITShaders.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.renderer.transparency;
+
+public final class OITShaders
+{
+
+    private OITShaders()
+    {
+        throw new IllegalStateException("Can not instantiate an instance of: ForgeTransparencyShaders. This is a utility class");
```

Comment:
See other statement on this.
It is recommended practice for library like systems to do this to prevent stupid reflection instantiation bugs, and is just part of my file template for utility classes.